### PR TITLE
[IMP] pos: remove duplicate of set customer and update sections

### PR DESCRIPTION
**Pull request by LoredanaLrpz**
task-3366190

### CODE REVIEW

File: content/applications/sales/point_of_sale/receipts_invoices.rst [hunk 1]
```diff
@@ -39,41 +39,36 @@ select the corresponding order and click :guilabel:`Print Receipt`.
...
-.. image:: receipts_invoices/invoice-button.png
-   :alt: button to generate an invoice in POS
+Upon processing a payment, click :guilabel:`Invoice` underneath the customer's name to issue an
+invoice for that order.
 
 Select the payment method and click :guilabel:`Validate`. The **invoice** is automatically issued
 and ready to be downloaded and/or printed.
 
+.. note::
+   A :ref:`customer <pos/customers>` must be selected to issue them an invoice.
```
Comment (chiaraprattico) [inline content/applications/sales/point_of_sale/receipts_invoices.rst:70]: To properly issue the :guilabel:`Invoice`, the customer must be selected.

**Review by chiaraprattico — APPROVED** [review]
I'm approving it only minor changes.

File: content/applications/sales/point_of_sale.rst [hunk 2]
```diff
@@ -51,14 +51,19 @@ Once an order is completed, proceed to checkout by clicking :guilabel:`Payment`.
...
 
-To access the list of customers, click :guilabel:`Customer` on the POS interface. Then, select a
-customer or create a new one by clicking :guilabel:`Create`, completing the form, and saving.
+You can create customers from an :ref:`open POS session <pos/session-start>` by clicking
+:menuselection:`Customer --> Create`, and filling in the contact information. You can also create
+customers from the backend by going to :menuselection:`Point of Sale --> Orders --> Customers` and
+clicking :guilabel:`New`. Then, fill in the information and save.
+
+To set a customer during an order, access the list of customers by clicking :guilabel:`Customer` on
+the POS interface. You can also set a customer at the payment screen by clicking
```
[the remark this answers is not in the document]
Comment (LoredanaLrpz) [reply content/applications/sales/point_of_sale.rst:65]: it is set as in "select" not configure :)

File: content/applications/sales/point_of_sale/receipts_invoices.rst [hunk 3]
```diff
@@ -39,41 +39,38 @@ select the corresponding order and click :guilabel:`Print Receipt`.
...
-In Point of Sale, you can issue and print invoices upon payment at the cash register or retrieve all
-the past invoiced orders.
+Point of Sale allows you to issue and print invoices for :ref:`registered customers <pos/customers>`
+upon payment and retrieve all past invoiced orders.
 
-Set a customer
---------------
+.. note::
+   An invoice created in a POS creates an entry into the corresponding :ref:`accounting journal
+   <cheat_sheet/journals>`, previously :ref:`setup <receipts_invoices/invoice_configuration>`.
```
Comment (xpl-odoo) [inline content/applications/sales/point_of_sale/receipts_invoices.rst:47]:
```suggestion
   <cheat_sheet/journals>` previously :ref:`set up <receipts_invoices/invoice_configuration>`.
```

**Review by xpl-odoo — APPROVED** [review]
Just one minor comment! :)
@robodoo delegate+

**Comment by LoredanaLrpz** [comment]
@robodoo r+

**Pull request closed without merging**